app/javascript/Pages/components/ButtonDifficultyFilter.scss

```diff
@@ -0,0 +1,15 @@
+.btn-difficulty {
```

Comment:
**suggestion (non-blocking)**: you don't need to create a file per component, all the buttons can be in the same file